eras/allegra/impl/cardano-ledger-allegra.cabal

```
@@ -1,6 +1,6 @@
 cabal-version:      3.0
 name:               cardano-ledger-allegra
-version:            1.0.0.0
+version:            1.0.0.1
```

Patch version is updated only when no semantic changes were introduced to the library, which is not the case here. We added a whole new publicly visible sublibray. Feel free to leave it as is, because I will be updating the major version in one of my PRs pretty soon anyways. I am just pointing it out that `d` in `a.b.c.d` is only bumped with changes in things like tests, documentation and potentially performance fixes.
```suggestion
version:            1.0.1.0
```

---

Got it! I will read up and refresh my understanding of the PVP.